packages/scripts/internationalization/common/version.js

```diff
@@ -36,7 +36,7 @@ function getPossibleVersion() {
 	}
 	if (!info && fs.existsSync(pomXmlPath)) {
 		const data = fs.readFileSync(pomXmlPath);
-		const { version } = xmlParser.toJson(data);
+		const { version } = JSON.parse(xmlParser.toJson(data)).project;
```

Comment:
why not use getPomXmlVersion ?

Reply:
`getPomXmlVersion ` was i fact a `getNextVersion`
But I did something else : a common function to read the xml